tests/test_perf_analyzer.py

```diff
@@ -141,6 +143,75 @@ def test_perf_analyzer_additive_args(self):
         with self.assertRaises(TritonModelAnalyzerException):
             self.config.to_cli_string()
 
+    def test_perf_analyzer_ssl_args(self):
```

Comment:
can you add a comment describing this function?

also this function and `test_perf_analyzer_ssl_options()` have very similar names. can the names be more clarifying?

Reply:
Updated the names and added a comment